app/views/ht_approval_requests/index.html.erb

```diff
@@ -21,13 +24,16 @@
 
     <% @reqs.each do |req| %>
       <tr <%= raw @added_users.include?(req.userid) || @renewed_users.include?(req.userid) ? 'class="success"' : '' %>>
-        <td>
-          <% if req.received.present? %>
-              <%= check_box_tag 'ht_users[]', req.userid %>
-            <% end %>
+        <% if controller.can?(:edit, :ht_approval_requests) %>
```

Comment:
I would personally prefer to see these view checks encoded in a presenter, though that can be pretty involved when first introducing them (especially with nested collections).

The CanCan(Can) way of doing this is a view helper to avoid the explicit target. I'm not especially fond of that, but it does provide an indirection that can be useful (and less verbose).

Reply:
For now I am decluttering by elevating can? to helper method status, and making the user param optional.

Reply:
Nice, I think that's a practical step.